core/src/main/scala/org/apache/spark/h2o/converters/H2ORDDLike.scala

```diff
@@ -23,7 +23,8 @@ import water.fvec.{Frame, FrameUtils}
 /**
  * Contains functions that are shared between all H2ORDD types (i.e., Scala, Java)
  */
-private[converters] trait H2ORDDLike[T <: Frame] {
```

Comment:
H2OContext was not moved. Its inner class was moved, as requested. The rest
is where it was.

Thanks,
-Vlad

On Thu, Aug 25, 2016 at 10:29 AM, Jakub Háva <notifications@github.com>
wrote:

> In core/src/main/scala/org/apache/spark/h2o/converters/H2ORDDLike.scala
> <https://github.com/h2oai/sparkling-water/pull/58#discussion_r76288310>:
>
> > @@ -23,7 +23,8 @@ import water.fvec.{Frame, FrameUtils}
> >  /**
> >   * Contains functions that are shared between all H2ORDD types (i.e., Scala, Java)
> >   */
> > -private[converters] trait H2ORDDLike[T <: Frame] {
>
> yup, I'm good with restriction like that, but still don't see why moving
> H2OContext somewhere. What are your intentions @vpatryshev
> <https://github.com/vpatryshev> with this please ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/58/files/dda8bacb29caf9d9ad724922e0bdf29e831b7ecb#r76288310>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK7X9ygjxdjR1SQUPOcUjlEfOjQYNks5qjdDsgaJpZM4Jsgi_>
> .
>